package.json

```diff
@@ -35,13 +35,13 @@
   "license": "MIT",
   "readmeFilename": "README.md",
   "devDependencies": {
-    "jshint": "~2.9.4",
+    "jshint": "~2.9.7",
```

Comment:
maybe do this in separate change if upstream want to pick this

Reply:
Sure. I'm reverting those changes. 

However, do you want to keep the jshint fixes? Basically I added some missing commas and renamed two variables that were still in scope due to JavaScript's variable hoisting. Since jshint is being run as a pretest step "npm test" was failing. That triggered my OCD and I had to fix it! If you think those changes is something upstream may be interested in adopting I'll keep them, otherwise we can drop them.